QUIZ.py

```diff
@@ -0,0 +1,46 @@
+
+
+### 8.  Create Contact.csv file which contain name,contact. Use "|" as Delimiter.
+###  Search perticular contact by entering person name . 
+
+#### creating contact.csv and take records.
+import csv
+f=open('c:\\22bca222\\python\\SQLITE3\\contact.csv', 'w',newline='')
+csv_w=csv.writer(f)
+column=['NAME','CONTACT']
+rec=[['om',9876576475],['sai',4534723545],['ram',7597863756],['yug',5465326556]]
+v_w.writerow(column)
+csv_w.writerows(rec)
```

Comment:
in above line you write writerow() and here you write writerows()..is it mistake? please tell me difference.

Reply:
No, it's not mistake.
In the above line i wrote writerow(column) that means it will write only one row  AND
in the second line i wrote writerows(rec) that means it will write multiple rows in the file.

Reply:
ok..!!! good.